client/src/components/Gallery.jsx

```diff
@@ -1,49 +1,44 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+
 function Gallery({
-  selectedStyle, displayIndex, incrementDisplayIndex, decrementDisplayIndex
+  selectedStyle, displayIndex, setDisplayIndex, incrementDisplayIndex, decrementDisplayIndex
 }) {
   let displaySlides = [];
   let displayThumbs = [];
   const pictureData = selectedStyle.photos;
   
-  function makeSlide(image) {
-    return (<img className='gallerySlide' src={image.url} alt={`${selectedStyle.name} style`} />);
-  }
-  
-  function makeThumbnailBar(image) {
-    return ((<li><img className='galleryThumbnail' src={image.thumbnail_url} alt={`${selectedStyle.name} style`} /></li>));
+  function scrollThumbnails() {
+    const thumbElement = document.querySelector('.galleryThumbView li');
+    thumbElement.style.marginLeft = thumbElement.scrollWidth * displayIndex * -1 + 'px';
   }
   
   function handleLeftArrowClick() {
-    const thumbElement = document.querySelector('.galleryThumbView li');
     if (displayIndex !== 0) {
       decrementDisplayIndex(displayIndex--);
     }
-    thumbElement.style.marginLeft = thumbElement.scrollWidth * displayIndex * -1 + 'px';
-    console.log(`Index: ${displayIndex} Pixel Math: ${-104 * displayIndex}, ThumbElementMarginLeft: ${thumbElement.style.marginLeft}`);   
+    scrollThumbnails();
   }
 
   function handleRightArrowClick() {
```

Comment:
I realize that these arrow click functions are repetitive code, and that other helper functions in this file could probably be defined outside of the main component. However I'm not sure what the most elegant way to manipulate the Redux Store are from outside the component.  Hooks?

Reply:
you can obtain values from the store using useSelector hooks

you can manipulate the state by using useDispatch (as long as the argument matches a reducer)